gammapy/utils/tests/test_nddata.py

```diff
@@ -0,0 +1,295 @@
+import numpy as np
```

Comment:
Add the usual two lines of boilerplate to every file: license and future imports.